solvers/CondatVu.py

```diff
@@ -69,3 +69,14 @@ def get_result(self):
     def st(self, w, mu):
         w -= np.clip(w, -mu, mu)
         return w
+
+    def get_l2norm(self, A, n_iter=100):
+        if isinstance(A, np.ndarray):
+            return np.linalg.norm(A, ord=2)
+        else:
+            AtA = A.T @ A
+            x = np.random.randn(AtA.shape[0])
+            for _ in range(n_iter):
+                x = AtA @ x
+                x /= np.linalg.norm(x)
+            return np.sqrt(np.linalg.norm(AtA @ x))
```

Comment:
this function should be put in a common utils folder like done here:

https://github.com/benchopt/benchmark_resnet_classif/tree/main/utils

Reply:
That is to say, to create a new folder ```utils``` and create a file ```l2-norm.py```?